package.json

```diff
@@ -59,7 +59,7 @@
     "light-bolt11-decoder": "^3.2.0",
     "lint-staged": "^14.0.0",
     "microbundle": "^0.15.1",
-    "nostr-tools": "^1.17.0",
+    "nostr-tools": "^2.10.4",
```

Comment:
is this supposed to be a dev dependency? but it seems to work

Reply:
Yeah same reaction, we only use it in examples, so just a simple change in here

Reply:
Ah, cool. Thanks!